libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -41,20 +42,33 @@ extern const AP_HAL::HAL& hal;
 // table of user settable CAN bus parameters
 const AP_Param::GroupInfo AP_UAVCAN::var_info[] = {
     // @Param: NODE
-    // @DisplayName: UAVCAN node that is used for Ardupilot
+    // @DisplayName: UAVCAN node that is used for this network
     // @Description: UAVCAN node should be set implicitly
     // @Range: 1 250
     // @User: Advanced
     AP_GROUPINFO("NODE", 1, AP_UAVCAN, _uavcan_node, 10),
 
+    // @Param: SRV_BM
+    // @DisplayName: RC Out channels to be transmitted as servo over UAVCAN
+    // @Description: Bitmask with one set for channel to be transmitted as a servo command over UAVCAN
+    // @Bitmask: 0: Servo 1, 1: Servo 2, 2: Servo 3, 3: Servo 4, 4: Servo 5, 5: Servo 6, 6: Servo 7, 7: Servo 8, 8: Servo 9, 9: Servo 10, 10: Servo 11, 11: Servo 12, 12: Servo 13, 13: Servo 14, 14: Servo 15
+    // @User: Advanced
+    AP_GROUPINFO("SRV_BM", 2, AP_UAVCAN, _servo_bm, 255),
```

Comment:
Not every, only first 8 :)
But yes, I tried to make it 'plug and play'. If it will be more correct to have it zero for default and manually enabled if needed - I do not mind at all.